packit_service/worker/handlers/distgit.py

```diff
@@ -531,6 +543,32 @@ def __init__(
     def get_checkers() -> Tuple[Type[Checker], ...]:
         return (ValidInformationForPullFromUpstream, IsUpstreamTagMatchingConfig)
 
+    def get_resolved_bugs(self) -> List[str]:
+        """
+        If we are reacting to New Hotness, return the corresponding bugzilla ID only.
+        In case of comment, take the argument from comment. The format in the comment
+        should be /packit pull-from-upstream --resolved-bugs rhbz#123,rhbz#124
+        """
+        if self.data.event_type in (NewHotnessUpdateEvent.__name__,):
+            bug_id = self.data.event_dict.get("bug_id")
+            return [f"rhbz#{bug_id}"]
+
+        comment = self.data.event_dict.get("comment")
+        commands = get_packit_commands_from_comment(
+            comment, self.service_config.comment_command_prefix
+        )
+        args = commands[1:] if len(commands) > 1 else ""
+        bugs_keyword = "--resolved-bugs"
+        if bugs_keyword not in args:
+            return []
+
+        bugs = (
+            args[args.index(bugs_keyword) + 1]
+            if args.index(bugs_keyword) < len(args) - 1
+            else None
+        )
```

Comment:
No need to do it now, but we should implement a generic way to work with arguments (maybe as part of #2200).

Maybe, we can really introduce argparse, or some other library as @mfocko has suggested..;)